index.js

```diff
@@ -135,7 +135,10 @@ function findGlobals(source) {
         return;
       }
     }
-    node.parents = parents;
+    node.parents = [];
+    for (var j = 0; j < parents.length; j++) {
+      node.parents.push(parents[j]);
+    }
```

Comment:
Can we just use `parents.slice()` to clone the array, rather than manually copying each value?